src/python/WMCore/Storage/Backends/XRDCPImpl.py

```diff
@@ -59,17 +101,44 @@ def _checkXRDUtilsExist(self):
 
         return foundXrdcp & foundXrdfs
 
-    def createStageOutCommand(self, sourcePFN, targetPFN, options=None, checksums=None):
+    def getAuthEnv(self, authMethod=None, forceMethod=False):
```

Comment:
Given that this method is not part of the interface - and is only used internally to this module - preceding it with an underscore (private method style) would look more clear.

Reply:
Done!